app/src/main/res/values/strings.xml

```diff
@@ -1387,7 +1387,7 @@
     <string name="summary_show_home_on_boot">Bring up xDrip screen automatically after device boots</string>
     <string name="title_show_home_on_boot">Show xDrip on Boot</string>
     <string name="eula">EULA</string>
-    <string name="summary_plus_accept_follower_actions">Treatments and Calibrations from followers will be accepted</string>
+    <string name="summary_plus_accept_follower_actions">Treatments, Calibrations and Sensor Stop from followers will be accepted</string>
```

Comment:
I think this should just say "other actions" instead of "Sensor Stop" as otherwise it just has to be re-translated if that functionality is ever extended - for example if starting sensor remotely was added.

Reply:
Done